game/src/ecs/components/ai/AITools.java

```diff
@@ -91,6 +91,12 @@ public static List<Tile> getAccessibleTilesInRange(Point center, float radius) {
         return tiles;
     }
 
```

Comment:
javadoc

Reply:
und dann bitte in `AiTools#getAccessibleTilesInRange` verwenden. Da frag ich auch nach allen Tiles und pick mir random eins raus.

Reply:
Du meinst vermutlich `AiTools#calculatePathToRandomTileInRange`. Hab's aber verstanden.